### ISSUE TITLE
[Feature Request] Remix Plugin for eth95

### ISSUE THREAD

**Issue by anudit** [adrianmcli/eth95#8]
How about a Remix Plugin that takes the deployed contract's Address, Network and ABI, and creates a query string that opens its eth95 link in a new tab. This can be used to directly open an eth95 UI for any contract from the Remix IDE. Sorta like https://github.com/One-Click-Dapp/remix-plugin-one-click-dapp.

**Comment by adrianmcli** [comment]
Right now the query string function works via Etherscan. Passing the ABI as a query string is doable, but it would take quite a bit more work unfortunately. I think this is a good idea, but just might make the code a little messy as it stands.

**Comment by anudit** [comment]
Makes sense. Thanks for considering, maybe sometime later down the line.
